build_modules/lib/src/module_builder.dart

```diff
@@ -37,60 +36,10 @@ class _CleanMetaModuleCache {
   }
 }
 
-/// Updates dependencies from the provided [Module] so that they point to the
-/// primary resource of the dependencies corresponding [Module].
-///
-/// Note that this will process the clean meta modules for the module
-/// dependencies if necessary.
-Future<Module> _cleanModuleDeps(
```

Comment:
I don't follow why this is no longer necessary. Can you quickly explain?

Reply:
This was all only used by the fine strategy fallback codepath - which no longer exists.